FinalEngine.Editor.Common/Services/Application/IApplicationContext.cs

```diff
@@ -7,23 +7,31 @@ namespace FinalEngine.Editor.Common.Services.Application;
 using System;
 
 /// <summary>
-/// Defines an interface that represents contextual information related to the current application.
+/// Defines an interface that represents contextual information related to the current application and it's associated data.
 /// </summary>
 public interface IApplicationContext
 {
+    /// <summary>
+    /// Gets the directory that serves as a common repository for Final Engine application-specific data for the current local user.
+    /// </summary>
+    /// <value>
+    /// The directory that serves as a common repository for application-specific data for the current local user.
+    /// </value>
+    string DataDirectory { get; }
+
     /// <summary>
     /// Gets the title of the application.
     /// </summary>
     /// <value>
-    /// The title of the application.
+    /// The title of the application, suffixed by the <see cref="Version"/>.
     /// </value>
```

Comment:
> **Note** Preference only!!

```suggestion
    /// <summary>
    /// Gets the title of the application.
    /// </summary>
```

I personally never use the `<value>` code doc.  The reason for this is because the end result always ends up being repetitive to the `<summary>` section.  All of the doc processing systems also work just fine without it.

For example, Velaptor does not use `<value>` at all and my API docs for [docs.velaptor.io](https://docs.velaptor.io) work just fine.  The end result would be that there is too much repetitive information, longer processing times, and any API doc sites would be **information overload** for the users.

This is of course not _**wrong**_ in any shape or form and is purely preference.  Just wanted to give you a heads up.

Reply:
I was thinking about this today and doing some research. I can see where `<value>` can have it's merits; such as a `bool?`. I tend to use `<remarks>` in cases where more information is needed, `GhostDoc` helps me out with documentation allot so maybe I should have a think about whether or not I should include the `<value>` tag or not.